File: FundamentosDeLaProgramacionLuisJoyanes4taEdicion/chapter05/CSHARP/ContinueSentence.cs
/*Libro:Fundamentos de Programación, Algoritmos, estructura de datos y objetos 4ta edición. 
    Capitulo 05.-Flujo de control II: Estructuras repetitivas
    Problema 10.-Sentecia continue. 
    Pagina 177
*/
using System;
class ContinueSentence{    
    public static void Main(string[] args){

           System.Console.Write("Cuenta del 1 - 25:\n\n");
            for(int i=1; i<=25; i++){

                if(i==5){
                    continue;
                }
                System.Console.Write(i+" ");
            }
            System.Console.Write("\n");
    }
}

File: Java2CursoDeProgramacion4aEdicion/javaConsole/chapter07/ContinueSentence.java
/**Libro: Java 2 Curso de programación 4a edición
*Capitulo 07: Sentencias de control
*Programa 11: Uso de la sentencia Continue. 
*/

public class ContinueSentence{
    public static void main(String[] args){
        for(int i=1; i<100; i++){
            
            if(i%5!=0){
                continue;
            }
            System.out.print(i+" ");
        }
        System.out.println();
    }
}